Automattic/wp-calypso
# Remove legacy passwordlessSignup A/B test and associated components and styles

This is a clean-up PR to remove the old passwordless sign up flow and A/B test since we haven't been using it for a long time (re: pbmo2S-Bv-p2#comment-1372)

#### Changes proposed in this Pull Request

* Remove `passwordlessSignup` A/B test since it is no longer used
* Also remove the separate `PasswordlessSignupForm` component
* Update a couple of style rules to make the linting happy

#### Testing instructions



* Make sure that creating an account via the `/start` flow works as expected.
* Double-check that I've removed all of this A/B test, and that I haven't removed anything that's needed elsewhere. Note: there is a passwordless component in the repo, that was never used for this A/B test so I've decided to keep removing it out of this PR to keep thing cleaner.

CC: @niranjan-uma-shankar and @taggon as I know you've both been working in the /start flow more than I have lately, just wanted to get a confidence check if either of you get the chance to take a look! I updated a tiny bit of CSS for the `reskinSignupFlow` to keep the linter happy, but I believe the text size is unchanged.

Fixes #
